tests/test_models/test_city.py

```diff
@@ -109,6 +109,6 @@ def test_to_dict_values(self):
 
     def test_str(self):
         """test that the str method has the correct output"""
-        city = City()
-        string = "[City] ({}) {}".format(city.id, city.__dict__)
-        self.assertEqual(string, str(city))
+        c = City()
+        string = "[City] ({}) {}".format(c.id, c.__dict__)
+        self.assertEqual(string, str(c))
```

Comment:
I'm assuming we made all of these 'city' to 'c' changes because of the unnecessary import on line 8. After removing line 8, I would prefer to revert the 'city' to 'c' changes for readability's sake.

Reply:
Yes, and to match previous codebase.  Let me change these back to city for readability.

Reply:
Thank you!